test-interface/src/main/scala/sbt/testing/Event.scala

```diff
@@ -1,41 +1,40 @@
 package sbt.testing
 
-/**
- * An event fired by the test framework during a run.
- */
+/** An event fired by the test framework during a run. */
 trait Event {
 
-  /**
-   * The fully qualified name of a class that can rerun the suite or test
-   * about which an event was fired.
+  /** The fully qualified name of a class that can rerun the suite or test
+   *  about which an event was fired.
```

Comment:
http://docs.scala-lang.org/style/scaladoc.html

But I am fine with how we do things. Just though that if there is a style ScalaStyle can enfore, we maybe should use it. But if it doesn't, we always have the option to write our own checker (I assume), if it becomes a problem.